File: server/aspnet/nvslonlineApi/nvslonlineApi/Models/Venue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace nvslonlineApi.Models
{
    public class Venue
    {
        public int Id { get; set; }

        public string VenueName { get; set; }

        public bool IsHidden { get; set; }
    }
}

File: client/angular2/NvslOnlineAngular2App/src/app/shared/models/venue.ts
import { IVenue } from '../interfaces/interfaces';

export class Venue implements IVenue{    
    
    constructor (public Id: number, public VenueName: string, public IsHidden: boolean) {
        this.Id = Id;
        this.VenueName = VenueName;        
        this.IsHidden = IsHidden;
    } 
}